axlearn/open_api/gemini.py

```diff
@@ -188,6 +187,24 @@ def _aggregate_tool_role_messages(messages: List[Dict[str, Any]]) -> List[Dict[s
     return aggregated_messages
 
 
+def _santize_request(request: Dict[str, Any]):
+    """Santizes request to follow Gemini request rules."""
+    if "messages" in request:
+        new_messages = []
+        for message in request["messages"]:
+            new_messages.append(_format_tool_message(message=message))
+        request["messages"] = new_messages
```

Comment:
nit -- could be simpler with list comprehensions.

Reply:
changed